spotbugs/src/main/java/edu/umd/cs/findbugs/detect/ThrowingExceptions.java

```diff
@@ -18,25 +29,73 @@ public ThrowingExceptions(BugReporter bugReporter) {
         this.bugReporter = bugReporter;
     }
 
+    private JavaClass klass;
```

Comment:
The usual convention is `clazz`.

Reply:
This conversation can be resolved, since it was fixed in b7d838e2efe50d68ccbe82953f0884543b5e0a2d.